clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -78,6 +83,18 @@ type Controller struct {
 	ssh util.SSHExecutor
 }
 
+// CompleteConfig completes Prometheus manifest file path config
+func CompleteConfig(p *config.PrometheusConfig) {
+	prefix := p.ManifestPath
+	if prefix == "" {
+		prefix = defaultPrometheusManifestPath
+	}
+	coreManifests = prefix + "/*.yaml"
+	defaultServiceMonitors = prefix + "/default/*.yaml"
+	masterIPServiceMonitors = prefix + "/master-ip/*.yaml"
+	nodeExporterPod = prefix + "/exporter/node-exporter.yaml"
```

Comment:
s/exporter/exporters/

[This is the test failure cause]

Reply:
Thanks!